sdk/storage/Azure.Storage.Common/tests/Shared/StorageRecordedTestSanitizer.cs

```diff
@@ -131,5 +131,18 @@ public override string SanitizeTextBody(string contentType, string body)
             // If anything goes wrong, don't sanitize
             return body;
         }
+
+        public override string SanitizeVariable(string variableName, string environmentVariableValue) =>
+            variableName switch
+            {
+                "Storage_TestConfigDefault" => SanitizeConnectionString(environmentVariableValue),
+                _ => base.SanitizeVariable(variableName, environmentVariableValue)
+            };
+
+        private static string SanitizeConnectionString(string connectionString)
+        {
+            connectionString = connectionString.Replace("AccountKey=Sanitized", "AccountKey=Kg==;");
```

Comment:
Our test recordings will now contain "Kg==;" instead of "Stanitized".  This is ok, because this value isn't checked by the recording framework.  This will allow us to record tests with connection strings.